src/components/pages/data/show/header/index.tsx

```diff
@@ -108,6 +135,28 @@ export default function ShowHeader() {
               ))}
             </Flex>
           )}
+          {isLoggedIn && !canEdit && (
+            <>
+              <Tooltip hasArrow label="Request Permission to Contibute">
+                <Button
+                  leftIcon={<MailIcon boxSize={"7"} />}
```

Comment:
In boxSize prop, instead of having a hard-coded number, can we have a relative value?